### DISCUSSION TITLE
Please read this, Very Important

### DISCUSSION THREAD

**Discussion by thesumedh** [community/community#112533] [GitHub Education]
### Select Topic Area

Question

### Body

I am a student currently pursuing a four year degree course in a Tier 3 college. My college doesn't have the infrastructure to provide email addresses to each of their students. Although GitHub says that they accept dated ID cards, I have been unsuccessful in getting approved despite providing the same. I have been trying for days now, and have spent hours today and uploaded every possible document including all my fees receipts, semester grade cards and dated ID card. GitHub just apologizes on submission and rejects my application without any proper reason or way to fix it. There are multiple community threads with people facing similar issues. Is there any other way for Tier 3 college students to verify themselves without university email addresses?
Is GitHub Student pack is really for student?
I'm trying since 6 months and rejected 13 times. It's very sad they are not fixed this...
![GithubSS](https://github.com/community/community/assets/118736444/86db05c0-eb0d-4941-a18b-5aa49babb078)

I want to learn AZURE, If this doesn't work I have to go with AWS, G cloud
very frustrated, Not expected from Microsoft/GitHub.

**Comment by thesumedh** [comment]
I'll wait for a few days and then decide whether to continue using Microsoft products or explore alternatives. Thanks

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @thesumedh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by thesumedh** [reply]
Tried but no one help. Im posting from last 6 Months.

**Comment by samus-aran** [comment]
Hi @thesumedh 👋 

What school do you attend? You need to ensure you select the correct one else you may submit incorrect evidence that would hinder your application. 

Check out the updated [Student Developer Pack Application FAQs & Common Rejection Reasons🎒](https://github.com/orgs/community/discussions/111352#top) to see if any of the answers assist you. Perhaps the following links will help:

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

**Reply by thesumedh** [reply]
I uploaded everything I had. Why don't you just accept that you only accept tier 1 and tier 2 college students or those from the American education system? If you have to scale, take feedback or check where students are stuck. Always giving the same reason seems robotic. I tried to chat with Azure guys, but they don't know much about this. Finally, I started learning AWS.

You need to check how many tier 3 college students benefit from the GitHub Student Developer Pack.
